index.html

```diff
@@ -0,0 +1,154 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
+  <title>Daa Nkwa</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="script.js"></script>
+</head>
+<body>
+  <header>
+    <!-- first navigation -->
+    <nav class="first-header">
+      <ul class="first-header-ul">
+        <li><a><img src="Img\facebook-letter-logo-svgrepo-com.svg" alt="Facebook"></a></li>
+        <li><a><img src="Img\twitter-white.svg" alt="twitter"></a></li>
+        <li>English</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+    </nav>
+    <!-- second second navigation -->
+    <nav class="nav-div">
+      <div class="logo">
+        <a href="index.html"><img class=logo src="Img\pngtree-blue-bird-vector-or-color-illustration-png-image_2013004.jpg"  alt="Logo"></a>
+      </div>
+      <ul class="nav justify-content-end">
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="index.html">Home</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link active" aria-current="page" href="about.html">About</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="index.html#program">Join</a>
```

Comment:
- [ ]  As previous reviewer suggested, I don't see the solution for the requested issue. You asked to avoid the blurred background and show the navigation with the right background from the the designed, but when I change into mobile screen all I see is the blurred background. please fix the issue for you to proceed 
![image](https://github.com/obibaadoma/captone_DaaNkwa/assets/43877689/48e2d28c-b1cb-4003-bb76-6306bf43195b)